### PR TITLE
docs: (Core) Add busy indicator to table documentation

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
part of https://github.tools.sap/dxp/jukebox/issues/442
#### Please provide a brief summary of this pull request.
There is added documentation for busy indicator usage with table
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

File: apps/docs/src/app/core/component-docs/table/examples/table-toolbar-example.component.html [hunk 1]
```diff
@@ -4,46 +4,55 @@
...
-        [glyph]="'decline'"
-        [placeholder]="'Search'"
-        [button]="true"
-        (addOnButtonClicked)="resetSearch()"
+            [compact]="true"
+            [disabled]="false"
+            [(ngModel)]="searchTerm"
+            (ngModelChange)="searchInputChanged($event)"
+            [glyph]="'decline'"
+            [placeholder]="'Search'"
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/table/examples/table-toolbar-example.component.html:12]: glyph="decline"
placeholder="Search"

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**